tests/phpunit/tests/test-wpml-cornerstone-register-strings.php

```diff
@@ -11,30 +11,52 @@ class Test_WPML_Cornerstone_Register_Strings extends OTGS_TestCase {
 	 * @test
 	 */
 	public function it_registers_strings() {
-		list( $name, $post, $package ) = $this->get_post_and_package( 'Cornerstone' );
+		list( , $post, $package ) = $this->get_post_and_package( 'Cornerstone' );
+
 		$string  = new WPML_PB_String( rand_str(), rand_str(), rand_str(), rand_str() );
-		$strings = array( $string );
-		$data    = array(
-			array(
-				'_type' => 'headline',
-				'field' => 'value',
-			),
-		);
-		$node_id = md5( serialize( $data[0] ) );
+		$strings = [ $string ];
+
+		$data = [
```

Comment:
@dgwatkins, in this `$data` array, you should also insert one module with the `classic:` prefix.
And you don't need a deep module, you can do this on the first dimension:

```php
$data = [
    [ // ... ],
    [
        '_type' => 'classic:section',
        '_modules' => [],
    ]
];
```

Reply:
A bug went by without noticing because we weren't using the deep structure.

If it's ok, I prefer to leave it like this.

Anyway, I added a "classic" module there